packages/zoe/src/contracts/sellItems.js

```diff
@@ -42,10 +43,19 @@ const start = zcf => {
   assertIssuerKeywords(zcf, harden(allKeywords));
   assertNatAssetKind(zcf, pricePerItem.brand);
 
+  /** @type {undefined | ZCFSeat} */
   let sellerSeat;
 
+  const { notifier: availableItemsNotifier,
+          updater: availableItemsUpdater } = makeNotifierKit();
+
   const sell = seat => {
     sellerSeat = seat;
+    observeIteration(sellerSeat.getNotifier(), harden({
+      updateState: sellerSeatAllocation => availableItemsUpdater.updateState(sellerSeatAllocation['Items']),
```

Comment:
Can we use dot notation instead? Same below for `finish`

```suggestion
      updateState: sellerSeatAllocation => availableItemsUpdater.updateState(sellerSeatAllocation.Items),
```